ios/FluentUI.Demo/FluentUIDemoTests/CardNudgeTest.swift

```diff
@@ -12,4 +12,29 @@ class CardNudgeTest: BaseTest {
     func testLaunch() throws {
         XCTAssertTrue(app.navigationBars[controlName].exists)
     }
+
+    func testElements() throws {
+        let mainIconSwitch: XCUIElement = app.tables.element(boundBy: 0).cells.element(boundBy: 2)
+        let subtitleSwitch: XCUIElement = app.tables.element(boundBy: 0).cells.element(boundBy: 3)
```

Comment:
Just for consistency, looks like we're testing all the switches except accent icon and accent. Should we test those too?

Also looks like we're testing just the main icon/subtitle switches in this method, whereas dismiss button/action button are in the other method. Maybe makes more sense if checking that all 6 switches work as expected happen in one method, while checking that actually clicking on the buttons work in the other method?

Reply:
The buttons have been very finicky so I only tested the reliable ones but I'll add them in to see if they run more reliably on Github.